readme.md

```diff
@@ -29,10 +29,19 @@ gulp.task('build', () => {
 
 This plugin will build the cordova project for the iOS platform.
 
-### Re-adding the iOS platform
+### Optional parameters
```

Comment:
Remove this entire section and add an `API` section like [this one](https://github.com/SamVerschueren/gulp-cordova-preference#api).